app/src/main/java/software/spartacus/com/shotclockscorekeeper/CoinFlipDialogFragment.java

```diff
@@ -48,6 +49,17 @@ public void onClick(DialogInterface dialog, int which) {
                 })
                 .create();
 
+        dialog.setOnShowListener(new DialogInterface.OnShowListener() {
+            @Override
+            public void onShow(DialogInterface dialogInterface) {
+                Button btnPositive = dialog.getButton(Dialog.BUTTON_POSITIVE);
+                btnPositive.setTextSize(40);
```

Comment:
I did not know you could do it that way.

Alternatively you can just use the dialog class and provide your own layout